cd/scripts/release-controller.sh

```diff
@@ -122,21 +121,42 @@ popd 1>/dev/null
 # build controller image
 if ! buildah bud \
   --quiet="$QUIET" \
-  -t "$AWS_SERVICE_DOCKER_IMG" \
+  -t "$AWS_SERVICE_DOCKER_IMG"-amd64 \
   -f "$CONTROLLER_IMAGE_DOCKERFILE_PATH" \
   --build-arg service_alias="$AWS_SERVICE" \
   --build-arg service_controller_git_version="$CHART_VERSION" \
   --build-arg service_controller_git_commit="$SERVICE_CONTROLLER_GIT_COMMIT" \
   --build-arg build_date="$BUILD_DATE" \
   --build-arg golang_version="$GOLANG_VERSION" \
-  --build-arg go_arch="$GOARCH" \
+  --build-arg go_arch="linux/amd64" \
+  --arch "amd64" \
   "$DOCKER_BUILD_CONTEXT"; then
   exit 2
 fi
 
+if ! buildah bud \
+  --quiet="$QUIET" \
+  -t "$AWS_SERVICE_DOCKER_IMG"-arm64 \
+  -f "$CONTROLLER_IMAGE_DOCKERFILE_PATH" \
+  --build-arg service_alias="$AWS_SERVICE" \
+  --build-arg service_controller_git_version="$CHART_VERSION" \
+  --build-arg service_controller_git_commit="$SERVICE_CONTROLLER_GIT_COMMIT" \
+  --build-arg build_date="$BUILD_DATE" \
+  --build-arg golang_version="$GOLANG_VERSION" \
+  --build-arg go_arch="linux/arm64" \
+  --arch "arm64" \
+  --variant v8 \
+  "$DOCKER_BUILD_CONTEXT"; then
+  exit 2
+fi
+
+buildah manifest create "$AWS_SERVICE_DOCKER_IMG"
+buildah manifest add --os=linux --arch=amd64 "$AWS_SERVICE_DOCKER_IMG" "$AWS_SERVICE_DOCKER_IMG"-amd64
+buildah manifest add --os=linux --arch=arm64 --variant v8 "$AWS_SERVICE_DOCKER_IMG" "$AWS_SERVICE_DOCKER_IMG"-arm64
+
 echo "Pushing '$AWS_SERVICE' controller image with tag: ${AWS_SERVICE_DOCKER_IMG}"
 
-if ! buildah push "${AWS_SERVICE_DOCKER_IMG}"; then
+if ! buildah manifest push --all "${AWS_SERVICE_DOCKER_IMG}" docker://"${AWS_SERVICE_DOCKER_IMG}"; then
```

Comment:
Do we also need to push the images? or pushing the manifest is enough here? /cc @acornett21

Reply:
@acornett21 
I use buildx on a regular basis, but am not that familiar with buildah.
I think the current code will work well enough.
Therefore, I would like to move forward with this feature.
If you feel that this code is insufficient, since it is OSS, may I ask you to create a PR for this PR of mine?
Thank you very much!

Reply:
By the way, I have tried to switch to jobs, but it did not work well because the parameters to be handed over are different for each architecture.

Reply:
@acornett21 Hello 👀

Reply:
Hi @kahirokunn, is there a question you are trying to ask?

Reply:
@acornett21 
This code works well enough.
Would you like to merge?

Reply:
@acornett21 
Do you have an opinion on this?
https://github.com/aws-controllers-k8s/test-infra/pull/383#discussion_r1337908302

Reply:
@a-hilaly 
This code works well enough.
Would you like to merge?

Reply:
Hi @kahirokunn I did not reply to comment 383 since you replied that you already tried to switch to jobs but were unsuccessful due to the different args that are passed in.

Reply:
Okay.